test/Test/Lynx/Form.purs

```diff
@@ -192,8 +198,10 @@ testTab :: String
 testTab = """
   { "name": "User"
   , "link": "user"
-  , "sections":
-    [ """ <> testSection <> """
+  , "contents":
+    [ { "tag": "Left"
```

Comment:
For consistency with other sum types we have, maybe we should make a data type for `TabContents _` and follow the pattern of stuffing a `type` field into the JSON object to distinguish the cases. How do you feel about that?

Reply:
That was my intention in the next iteration (you mentioned the utility of controlling the encoding while we were working on it 😄). I plan to write something similar to what I did in #35.